### PR TITLE
Updated resources.md

### PR DESCRIPTION
**Pull request by alekhgupta1441**

### CODE REVIEW

**Comment by aSquare14** [comment]
Please correct formating. And do not remove past resources.

**Comment by sourabh1031** [comment]
@alekhgupta1441 first and foremost this is not a good etiquette to make pr as i was working on issue . Please give me little time don't be hurry . I think you are a beginner

**Comment by aSquare14** [comment]
@sourabh1031 This issue is pretty open ended. I don't mind if many people work on it. So @alekhgupta1441 please feel free to work. :)

**Comment by aSquare14** [comment]
You need to rebase from master.

File: resources.md [hunk 1]
```diff
@@ -11,6 +11,17 @@ They have been arranged alphabetically.
...
+- [Explain git Like I’m Five](https://hackernoon.com/understanding-git-fcffd87c15a3)
+- [Git Handbook](https://guides.github.com/introduction/git-handbook/)
+- [Git Learn git, faster version control](https://www.tutorialspoint.com/git/)
+- [git - the simple guide](http://rogerdudler.github.io/git-guide/)
+- [Git Learn git, faster version control](https://www.tutorialspoint.com/git/)
+- [git - the simple guide](http://rogerdudler.github.io/git-guide/) 
+- [Git Tutorial Learn Git, Fast Version control](https://www.tutorialspoint.com/git/)
+- [Git Official Document](https://git-scm.com/docs/user-manual.html)
+- [Learn Git Branching](https://learngitbranching.js.org/)
+=======
```
Comment (vibhavagarwal5) [inline resources.md]: Can u pls remove this?

**Pull request closed without merging**